pom.xml

```diff
@@ -1452,7 +1452,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- Carbon kernel version -->
-        <carbon.kernel.version>4.4.23</carbon.kernel.version>
+        <carbon.kernel.version>4.4.27-SNAPSHOT</carbon.kernel.version>
```

Comment:
Is there any specific reason to change the kernel version to a SNAPSHOT version?

Reply:
As mentioned in the PR descrription, this PR should be merged after wso2/carbon-kernel#1737. This depends on kernel upgrade. After the merge, we can update the PR with released version